packages/framework/test-client-utils/src/ConsoleLogger.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryBaseLogger, ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+
+// Basic implementation of an ITelemetry Logger. This logger will be passed into a client constructor
+// and gets hooked up to the container telemetry system.
```

Comment:
/**
 * Basic implementation of an ITelemetry Logger. 
 * This logger will be passed into a client constructor and gets hooked up to the container telemetry 
 * system.
 */